### PR TITLE
tried to implement todo list add item

### PR DESCRIPTION
**Pull request by pedrosilva410**
needs work

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,20 @@
-const date = new Date()
-const todayDate = date.toLocaleDateString('de-de');
-document.getElementById("todayDate").innerHTML = todayDate;
+const date = new Date();
+const todayDate = date.toLocaleDateString();
+document.getElementById('todayDate').innerHTML = todayDate;
 console.log(todayDate);
```
Comment (201flaviosilva) [inline src/index.js]: remove this line

File: index.html [hunk 2]
```diff
@@ -12,15 +12,15 @@
 	<header>
 		<div class="title-date">
 			<h1>Todo List</h1>
-			<p id="todayDate">date</p> <!--this will display the date-->
+			<time id="todayDate"></time>
 		</div>
 		<div class="input">
```
Comment (201flaviosilva) [inline index.html:17]: This should be a form

File: src/index.js [hunk 3]
```diff
@@ -1,4 +1,20 @@
...
+const userInput = document.getElementById('userInput');
+const addButton = document.getElementById('addButton');
+const todoList = document.getElementById('todoList');
+
+addButton.addEventListener(click, addItem);
+
+function addItem() {
+	const newItem = document.createElement('li');
+	const newText = document.createElement('p');
+	const newInput = userInput.innerHTML();
```
Comment (201flaviosilva) [inline src/index.js]:
```suggestion
	const newInput = userInput.value;
```

File: index.html [hunk 4]
```diff
@@ -12,15 +12,15 @@
...
-			<input type="text" id="userInput" placeholder="Write down your goals!">
+			<input id="userInput" placeholder="Write down your goals!">
 			<button id="addButton">+</button>
```
Comment (201flaviosilva) [inline index.html]: This button should be a submit button
```suggestion
			<button type="submit" id="addButton">+</button>
```

File: src/index.js [hunk 5]
```diff
@@ -1,4 +1,20 @@
...
+const date = new Date();
+const todayDate = date.toLocaleDateString();
+document.getElementById('todayDate').innerHTML = todayDate;
 console.log(todayDate);
+
+const userInput = document.getElementById('userInput');
+const addButton = document.getElementById('addButton');
+const todoList = document.getElementById('todoList');
+
+addButton.addEventListener(click, addItem);
```
Comment (201flaviosilva) [inline src/index.js]: This event should be in the form, on submit: https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event

File: src/index.js (hunk 1 above)
Comment (pedrosilva410) [reply src/index.js]: the log?

File: index.html (hunk 2 above)
Comment (pedrosilva410) [reply index.html:17]: ok

File: src/index.js (hunk 3 above)
Comment (pedrosilva410) [reply src/index.js]: ok

File: src/index.js (hunk 1 above)
Comment (201flaviosilva) [reply src/index.js]: yap

**Pull request merged**